bitrise.yml

```diff
@@ -13,25 +13,18 @@ app:
   - SLACK_CHANNEL: "..."
   - opts:
       is_expand: false
+    FASTLANE_XCODEBUILD_SETTINGS_TIMEOUT: '1000'
```

Comment:
neni to tu mač? :)

Reply:
Na tomto kroku se to obvykle na našich projektech nezakne z jiného důvodu, než instalacka balíčků. Tak těch 15 minut snad neuškodí.